packages/pubspec.yaml

```diff
@@ -0,0 +1,14 @@
+name: devtools
```

Comment:
this seems odd to create a new pubspec for a directory that isn't really supposed to be a a dart package. Why are we adding this?

Reply:
It was advised by analyzer people. But I could remove it.